source/documentation/tools/index.md

```diff
@@ -0,0 +1,34 @@
+# Tools
```

Comment:
I'm not sure whether moving the guidance for RStudio, Jupyter and Airflow into a Tools section makes in easier to navigate than having them at the top level. Might be worth getting some feedback from Alex or Sam on this?

Reply:
I was thinking the same thing, however i thought that was one refactor too many for this ticket

Reply:
Ok cool, we can always re-jig if needed.